src/components/ContactMe/ContactInfoCard/index.jsx

```diff
@@ -1,17 +1,19 @@
-import React, { useState } from "react";
+import React, { useState, useEffect } from "react";
 import "./styles.css";
 import Text from "../../Text";
 import Icon from "../../Icon";
 
 const ContactInfoCard = ({ text, name, link, linkLabel, showCopy }) => {
   const [isCopied, setIsCopied] = useState(false);
-
+  useEffect(() => {
+    const timer = setTimeout(() => setIsCopied(false), 2000); // Hide message after 2 seconds
```

Comment:
why do you need to put setTimeout here? Think twice, try to learn and understand how it works than apply.

Reply:
I thought that the component might be unmounted while setTimeout is working and I wrote the following code, to avoid it using useRef and useEffect, check it when you are free

Reply:
this will do the following:

It will make setIsCopied to false after 2s when you do page reload. but you need it to be false when you COPY the text right? So this setTimeout must be in a place where you copy your text and set it true, then after 2s it will be false again, but you need to clear timeout inside the useEffect to avoid memory leak.